drafts/use-cases/5.creating-new-event.md

```diff
@@ -22,25 +23,26 @@ var client = new HydraClient();
 var operation = client.get("http://example.com")
     .getApiDocumentation()
     .getEntryPoint()
-    .getOperations()
-    .filter(op => op.method == "POST" && op.expects == "Event")[0];
+    .onLink('#events')
```

Comment:
> I'm not sure what onLink('#events') means.

From EntryPoint client should follow the `#events` *hydra:Link* (again, which definition I could move to JSON-LD context of this API instead of using fragment) which pints to the collection of events and we want to get particular operation on that collection resource.